src/Transition/CreateNewIDFUsingRulesV24_2_0.f90

```diff
@@ -485,6 +570,95 @@ SUBROUTINE CreateNewIDFUsingRules(EndOfFile,DiffOnly,InLfn,AskForInput,InputFile
 
               ! If your original object starts with Z, insert the rules here
 
+!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+              CASE('ZONEHVAC:TERMINALUNIT:VARIABLEREFRIGERANTFLOW')
+                isVariableVolume = .FALSE.
+                CALL GetNewObjectDefInIDD(ObjectName, NwNumArgs, NwAorN, NwReqFld, NwObjMinFlds, NwFldNames, NwFldDefaults, NwFldUnits)
+                nodiff = .false.
+                OutArgs(1:13) = InArgs(1:13)
+                IF (SameString(InArgs(14), 'FAN:VARIABLEVOLUME')) THEN
+                  isVariableVolume = .TRUE.
+                  OutArgs(14) = 'Fan:SystemModel'
+                  OutArgs(15) = TRIM(InArgs(15))
+                  sysFanName = TRIM(InArgs(15))
+                ELSE
+                  OutArgs(14:15) = InArgs(14:15)
+                ENDIF
+                OutArgs(16:CurArgs) = InArgs(16:CurArgs)
+                CALL WriteOutIDFLines(DifLfn, 'ZoneHVAC:TerminalUnit:VariableRefrigerantFlow', CurArgs, OutArgs, NwFldNames, NwFldUnits)
+
+                IF (isVariableVolume) THEN
+                  ! create fan system model object
+                  ObjectName = 'Fan:SystemModel'
+                  DO Num3 = 1, NumFanVariableVolume
+                    IF (SameString(OldFanVO(Num3)%oldFanName, sysFanName)) THEN
+                      CALL GetNewObjectDefInIDD(ObjectName, NwNumArgs, NwAorN, NwReqFld, NwObjMinFlds, NwFldNames, NwFldDefaults, NwFldUnits)
+                      OutArgs(1) = TRIM(sysFanName)
+                      OutArgs(2) = OldFanVO(Num3)%availSchedule
+                      OutArgs(3) = OldFanVO(Num3)%inletAirNodeName
+                      OutArgs(4) = OldFanVO(Num3)%outletAirNodeName
+                      OutArgs(5) = OldFanVO(Num3)%maxAirFlow_str
+                      OutArgs(6) = 'Continuous'                             !- Speed Control Method
+                      IF (SameString(OldFanVO(Num3)%minFlowInputMethod, "FixedFlowRate")) THEN
+                        OutArgs(7) = OldFanVO(Num3)%maxAirFlow_str
```

Comment:
This should be 
`OutArgs(7) = OldFanVO(NumOldFanVO)%fanPowerMinAirFlow_str / OldFanVO(Num3)%maxAirFlow_str`